spec/requests/care_plan_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe "Care Plan", type: :request do
+  describe '#index' do
+    context 'when user is signed in' do
+      let(:user) { create(:user) }
+      before do
+        sign_in user
+        get '/care_plan'
+      end
+
+      it 'renders template' do
+        expect(response).to render_template(:index)
+      end
+
+      it 'does not render a different template' do
+        expect(response).to_not render_template(:show)
+      end
+
+      it 'response is successful' do
+        expect(response).to have_http_status(:ok)
+        expect(response.content_type).to eq('text/html; charset=utf-8')
+        expect(response).to be_successful
+      end
+
+      context 'when there are bookmared strategies' do
+        let(:strategy) { create(:strategy, user: user, bookmarked: true) }
+
+        it 'assigns @bookmarked_strategies' do
+          get '/care_plan'
```

Comment:
try to keep the code consistent. If you're going to use `before { get '/care_plan' } ` everywhere else in the test, also use it here

Reply:
I'd recommend you use route helpers instead. You can check out the available options via `bundle exec rake routes`. 

Let's use `get care_plan_path` instead

Reply:
it's solved, thank you